scripts/qa_crowbarsetup.sh

```diff
@@ -3392,9 +3447,8 @@ EOF
         test $cephret -eq 0 || ret=104
     fi
 
-    if [ "$wanttempest" = "1" ]; then
-        scp $novacontroller:/var/lib/openstack-tempest-test/tempest.log .
-        scp $novacontroller:/var/lib/openstack-tempest-test/tempest.subunit.log .
+    if [[ $wanttempest ]] ; then
+        scp $novacontroller:/var/lib/openstack-tempest-test/tempest*.log .
```

Comment:
I would like to add a comment here: in the original code the `scp` to tempest.subunit.log is explicit, to signalize that if after some refactoring this file is missing or renamed, something is wrong.

This file is used by this Rally/Flask extension used in QA to monitor the tempest evolution, so is difficult to notice when this file is not present, making an external service fail.